diff --git a/node_modules/react-native-login-screen/build/dist/LoginScreen.js b/node_modules/react-native-login-screen/build/dist/LoginScreen.js
index a7d879e..0981f48 100644
--- a/node_modules/react-native-login-screen/build/dist/LoginScreen.js
+++ b/node_modules/react-native-login-screen/build/dist/LoginScreen.js
@@ -13,13 +13,13 @@ const dummyFunction = () => { };
 const LoginScreen = ({ style, dividerStyle, logoImageStyle, loginTextStyle, loginButtonStyle, haveAccountTextStyle, haveAccountButtonStyle, textInputContainerStyle, haveAccountText = "Already have an account?", disableDivider, logoImageSource, onLoginPress, disableSocialButtons, onHaveAccountPress, onEmailChange, onPasswordChange, onFacebookPress = dummyFunction, onTwitterPress = dummyFunction, onApplePress = dummyFunction, onDiscordPress = dummyFunction, children, }) => {
     const Logo = () => (<react_native_1.Image resizeMode="contain" source={logoImageSource} style={[LoginScreen_style_1.default.logoImageStyle, logoImageStyle]}/>);
     const TextInputContainer = () => (<react_native_1.View style={[LoginScreen_style_1.default.textInputContainer, textInputContainerStyle]}>
-      <react_native_text_input_interactive_1.default placeholder="Email" onChangeText={onEmailChange}/>
+      
       <react_native_1.View style={LoginScreen_style_1.default.passwordTextInputContainer}>
-        <react_native_text_input_interactive_1.default placeholder="Password" secureTextEntry onChangeText={onPasswordChange}/>
+        <react_native_text_input_interactive_1.default placeholder="Mật khẩu" secureTextEntry onChangeText={onPasswordChange} color="black" />
       </react_native_1.View>
     </react_native_1.View>);
     const LoginButton = () => (<react_native_1.TouchableOpacity style={[LoginScreen_style_1.default.loginButtonStyle, loginButtonStyle]} onPress={onLoginPress}>
-      <react_native_1.Text style={[LoginScreen_style_1.default.loginTextStyle, loginTextStyle]}>Login</react_native_1.Text>
+      <react_native_1.Text style={[LoginScreen_style_1.default.loginTextStyle, loginTextStyle]}>Đăng nhập</react_native_1.Text>
     </react_native_1.TouchableOpacity>);
     const AlreadyHaveAccount = () => (<react_native_1.TouchableOpacity style={[LoginScreen_style_1.default.haveAccountButtonStyle, haveAccountButtonStyle]} onPress={onHaveAccountPress}>
       <react_native_1.Text style={[LoginScreen_style_1.default.haveAccountTextStyle, haveAccountTextStyle]}>
